### PR TITLE
[EN-913] CultureInfo.GetCultureInfoByIetfLanguageTag

### PR DESCRIPTION
**Pull request by izhemlunalabs**

### CODE REVIEW

File: Bridge/Resources/Globalization.js [hunk 1]
```diff
@@ -327,23 +330,57 @@
...
-                } else if (name === "") {
+                }
+                if (name === "") {
                     return System.Globalization.CultureInfo.invariantCulture;
                 }
-                var lowerName = name.toLowerCase();
-                var names = Bridge.getPropertyNames(this.cultures);
+                const names = Bridge.getPropertyNames(this.cultures);
+                // match always exists here because we`ve validated the name before
+                const leftTagInput = name.match(this.LEFT_TAG_REGEXP)[0].toLowerCase();
```
Comment (yard) [inline Bridge/Resources/Globalization.js:362]: What if ```name``` doesn't match a regex? It's gonna crash as ```match``` will return null

File: Bridge/Resources/Globalization.js [hunk 2]
```diff
@@ -327,23 +330,57 @@
...
                     }
                 }
 
-                throw new System.Globalization.CultureNotFoundException.$ctor5("name", lowerName);
+                // clone invariant culture with provided name
+                let newCultureName;
+                if (name.length === 2) {
+                    newCultureName = name.toLowerCase();
+                } else {
+                    newCultureName = name.substring(0, 3).toLowerCase() + name.substring(3, 5).toUpperCase();
```
Comment (yard) [inline Bridge/Resources/Globalization.js:378]: Looks a bit dirty to me. Can we use a regex with two capturing groups for better reliability?
const chunks = name.match(/^(\w\w).?(\w\w)?/);
```newCultureName = chunks[1].toLowerCase() + (chunks[2] ? "_" + chunks[2].toUpperCase() : "")```;

File: Bridge/Resources/Globalization.js (hunk 1 above)
Comment (izhemlunalabs) [reply Bridge/Resources/Globalization.js:362]: I put validation on the top of this method. And comment about it as well :)

File: Bridge/Resources/Globalization.js (hunk 2 above)
Comment (izhemlunalabs) [reply Bridge/Resources/Globalization.js:378]: Bigger RegExp has no match if name is "xy" only. Will keep length-check.

**Pull request merged**